apps/condo/domains/user/components/containers/PosterLayout.tsx

```diff
@@ -47,19 +58,48 @@ export const PosterLayout: React.FC<IPosterLayoutProps> = ({ children, headerAct
                 </Col>
                 <Col span={24}>
                     <Footer isSmall={isSmall} >
-                        <Typography.Link
-                            href={`mailto:${SUPPORT_EMAIL}`}
-                            style={TYPOGRAPHY_CONTACT_STYLE}
-                        >
-                            {SUPPORT_EMAIL}
-                        </Typography.Link>
-                        {', '}
-                        <Typography.Link
-                            href={`tel:${SUPPORT_PHONE}`}
-                            style={TYPOGRAPHY_CONTACT_STYLE}
-                        >
-                            {SUPPORT_PHONE}
-                        </Typography.Link>
+                        <Row style={FOOTER_ROW_STYLE}>
+                            <Row>
+                                <Typography.Link
+                                    href={`mailto:${SUPPORT_EMAIL}`}
+                                    style={TYPOGRAPHY_CONTACT_STYLE}
+                                >
+                                    {SUPPORT_EMAIL}
```

Comment:
Is it working ok if env is not specified?

Reply:
no